### DISCUSSION TITLE
Command Line support please

### DISCUSSION THREAD

**Discussion by LiveFreeDead** [mjishnu/alt-app-installer#54] [Ideas]
I would like to be able to run the .exe with a .appxbundle and have it silently install. even if it needs a switch, I'd be happy.

**Comment by mjishnu** [comment]
i was also planning on implementing this in future. i am in a process of rewriting the entire code base to c#, i will try to implement this there. currently its in very initial stage i will provide the repo link after its some what done.

**Comment by mjishnu** [comment]
here is the cli version https://github.com/mjishnu/alt-app-installer-cli
